ECLIPs/ECIP-draft_succinct-proofs.md

```diff
@@ -0,0 +1,109 @@
+---
+eip: <to be assigned>
```

Comment:
```suggestion
ecip: 1055
```

why did you use the EIP template not the ECIP template?

Reply:
my thought was that ETH might want to do it too, and that I would submit it as both EIP and ECIP